File: AMS.Web/src/app/models/geography/geography-response.ts
import { State } from "./state";
import { City } from "./city";
import { OperationStatus } from "../shared/operation-status";

export class GeographyResponse 
{
    //#region Constructor
    
    public GeographyResponse()
    {
        this.States = new Array<State>();
        this.Cities = new Array<City>();

        this.OperationStatus = new OperationStatus();
    }
    //#endregion Constructor

    //#region Class Properties

    States: Array<State>;

    Cities:Array<City>;

    OperationStatus: OperationStatus

    //#endregion
}

File: AMS.API/DTO/Geography/GeographyResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.Geography
{
    public class GeographyResponse
    {
        #region Constructor

        public GeographyResponse()
        {
            States = new List<State>();
            Cities = new List<City>();

            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<State> States { get; set; }

        public List<City> Cities { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}